pyfatx/__init__.py

```diff
@@ -148,6 +148,9 @@ def read(self, path: str, offset: int = 0, size: Optional[int] = None) -> bytes:
 		path = self._sanitize_path(path)
 		attr = self.get_attr(path)
 		assert(attr.is_file)
+		if (offset == 0 and attr.file_size == 0):
+			return bytes()
```

Comment:
Nit, it uses `b''` on line 158, should probably be consistent in how we return empty buffers.

Reply:
Fixed